Mathlib/Probability/Kernel/Disintegration.lean

```diff
@@ -369,6 +369,32 @@ theorem kernel.const_eq_compProd (γ : Type*) [MeasurableSpace γ] (ρ : Measure
     kernel.ext_iff'.mp (kernel.const_unit_eq_compProd ρ) () s hs
 #align probability_theory.kernel.const_eq_comp_prod ProbabilityTheory.kernel.const_eq_compProd
 
+/-- If the singleton `{x}` has non-zero mass for `ρ.fst`, then for all `s : Set Ω` measurable,
+`ρ.condKernel x s = (ρ.fst {x})⁻¹ * ρ ({x} ×ˢ s)` . -/
+lemma condKernel_apply_of_ne_zero [MeasurableSingletonClass α] {x : α} (hx : ρ.fst {x} ≠ 0)
```

Comment:
ρ is explicit in this lemma, I would probably make it implicit as it can be deduced both from hx and from either side of the conclusion.